conbench/entities/benchmark_result.py

```diff
@@ -391,8 +403,27 @@ class _BenchmarkFacadeSchemaCreate(marshmallow.Schema):
         },
     )
     batch_id = marshmallow.fields.String(required=True)
-    timestamp = marshmallow.fields.DateTime(
-        required=True, metadata={"description": "Timestamp the benchmark ran"}
+
+    # `AwareDateTime` with `default_timezone` set to UTC: naive datetimes are
+    # set this timezone.
+    timestamp = marshmallow.fields.AwareDateTime(
+        required=True,
+        format="iso",
+        default_timezone=timezone.utc,
+        metadata={
+            "description": conbench.util.dedent_rejoin(
+                """
+                A datetime string indicating the time at which the benchmark
+                was started. Expected to be in ISO 8601 notation.
+                Timezone-aware notation recommended. Timezone-naive strings are
+                interpreted in UTC. Fractions of seconds can be provided but
+                are not returned by the API. Example value:
+                2022-11-25T22:02:42Z. This timestamp defines the default
+                sorting order when viewing runs via the UI or when enumerating
+                runs via the /api/runs/ HTTP endpoint.
```

Comment:
A note on this. For many years I have `Alt +Q` in my editor bound to doing `rewrap` (rewrap text so that it nicely fills up to N columns). The triple-quoted strings in Python allow for using that. But they contain literal newlines, and if indention is applied that there also is some leading whitespace. Those additional newlines and the indention can of course be removed with programmatic tools.